src/main/webapp/login/container.js

```diff
@@ -16,9 +16,8 @@ const useLogin = () => {
 }
 
 const setCookie = cookie => {
-  let parsedCookie = cookie.split(';')
-  parsedCookie = parsedCookie[0].split('=')
-  cookies.set(parsedCookie[0], parsedCookie[1], { path: '/' })
+  const parsedCookie = cookie.split(';')
```

Comment:
I think the cookie parsing and encoding should happen on the graphql side not the client side.